src/prebuilt/wasm-ast-parser-gen.c

```diff
@@ -146,6 +146,7 @@
       APPEND_FIELD_TO_LIST(module, export_field, EXPORT, export_, loc_, \
```

Comment:
Yeah, I'd rather not, but it makes it easier for developers who don't have bison + re2c to compile. By default it is not used.